tests/dragonfly/snapshot_test.py

```diff
@@ -217,6 +217,38 @@ async def test_snapshot(self, df_seeder_factory, async_client, df_server):
         assert super().get_main_file("test-set-snapshot_cron-summary.dfs")
 
 
+@dfly_args({**BASIC_ARGS})
+class TestDoubleSave(SnapshotTestBase):
+    """Test simultaneous save operations"""
+
+    @pytest.fixture(autouse=True)
+    def setup(self, tmp_dir: Path):
+        super().setup(tmp_dir)
+
+    @pytest.mark.asyncio
+    @pytest.mark.slow
+    async def test_snapshot(self, df_seeder_factory, async_client, df_server):
+        seeder = df_seeder_factory.create(
```

Comment:
I think you should reduce the number of keys here as it will take very long to run this test in regression tests.
Maybe also worth using debug populate here to make this faster as working with seeder here does not adds any value

Reply:
Oh, missed this, 100k will take _eternity_ on CI 😆